src/cares_wrap.h

```diff
@@ -27,7 +27,7 @@
     defined(__OpenBSD__) || \
     defined(_MSC_VER)
 
-# include <nameser.h>
+# include <ares_nameser.h>
 #else
 # include <arpa/nameser.h>
 #endif
```

Comment:
I think this entire `#if defined` can be replaced with `#include <ares_nameser.h>` since it also conditionally includes `<arpa/nameser.h>`:
https://github.com/nodejs/node/blob/668af8113534a47cdf183649bcda2c6c71b06712/deps/cares/src/lib/ares_nameser.h#L5-L7